examples/cosmos3_story_scene/app/story_scene/config.py

```
@@ -0,0 +1,162 @@
+
+
+@dataclass(frozen=True, slots=True)
+class AppConfig:
+    """All browser-process settings; job submissions cannot override these."""
+
+    peer_host: str
+    peer_user: str = getpass.getuser()
+    ssh_key: Path | None = None
+    known_hosts: Path | None = None
+    work_root: Path = DEFAULT_WORK_ROOT
+    remote_root: str = "/var/tmp/cosmos3-physics-dual-spark"
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/var/tmp/cosmos3-physics-dual-spark"

(S108)